src/wordpress-playground/worker-thread.ts

```diff
@@ -176,6 +176,11 @@ class WordPressPatcher {
 			`${DOCROOT}/wp-content/mu-plugins/add_requests_transport.php`,
 			require('./mu-plugins/add_requests_transport.php')
 		);
+		// Various random hacks
+		this.#php.writeFile(
+			`${DOCROOT}/wp-content/mu-plugins/my-hacks.php`,
+			require('./mu-plugins/my-hacks.php')
```

Comment:
I'd pick a filename that communicates the purpose of that script, e.g. `1-show-admin-credentials-on-wp-login.php`. The next feature would go to another file, and so on. Collective names like `utils` or `hacks` may be a bit quicker to come up with today, but they bite hard when it's time to debug something later on.

Reply:
Sounds good, updated with da42674

Here's the history of `my-hacks.php`: https://make.wordpress.org/core/2015/09/18/my-hacks-php-no-longer-supported/

It was meant to be a bit of an easter egg 😁